mogc_partnerships/views.py

```diff
@@ -30,6 +33,8 @@
 from .pagination import LargeResultsSetPagination
 from .permissions import ManagerCreatePermission, ManagerEditPermission
 
+logger = logging.getLogger(__name__)
```

Comment:
Should we be using this `logger` somewhere?

Reply:
It's used here: https://github.com/academic-innovation/mogc-partnerships/pull/44/files#diff-cb8ad9970d804f3eb64f018098560f578443535b73ff67dbe852b363af8063d0R207

Reply:
Thanks! Not sure how I missed it.